.github/workflows/ci.yml

```diff
@@ -136,8 +140,7 @@ jobs:
             --openssl \
             --njs \
             --cc-opt="-I njs/src/ -I njs/build"  \
-            --ld-opt="-L njs/build" \
-            --debug
+            --ld-opt="-L njs/build"
 
       - name: Make unit
         run: |
```

Comment:
Looks good. But perhaps this should be split out into own commit separate from the language updates...

Reply:
I'm messing around with it to force a failure, but ya I agree! I'll undraft this when it's good to go.